modules/doc/content/newsletter/2022_02.md

```diff
@@ -7,6 +7,56 @@ for a complete description of all MOOSE changes.
 
 ## MOOSE Improvements
 
+### Lagrangian solid mechanics kernels
+
+A new system of solid mechanics kernels are available in the
+[tensor mechanics](modules/tensor_mechanics/index.md) module.
+These kernels both duplicate the existing updated Lagrangian formulation
+used in the current kernels and also add a new option for a total 
+Lagrangian formulation.  The main goal of this project was to 
+provide exact large deformation Jacobians without requiring the use of
+AD to increase the efficiency of running large, finite deformation simulations.
+The system 
+[documentation](modules/tensor_mechanics/LagrangianKernelTheory.md)
+includes a detailed description of the mathematical theory underlying the
+kernels as well as information on the new features available along
+with the kernels themselves.
+
+Some of these new features include:
+
+* Cartesian versions of a reformulated [updated Lagrangian](source/kernels/lagrangian/UpdatedLagrangianKernel.md) kernel and a new 
+[total Lagrangian](source/kernels/lagrangian/TotalLagrangianKernel.md) kernel, both with exact large deformation Jacobians.
+* [Stabilization](modules/tensor_mechanics/Stabilization.md) for linear triangular and tetraheadral elements.
+* A [new material system](modules/tensor_mechanics/NewMaterialSystem.md), which provides new options for implementing 
+material models, including options for defining material models
+with alternative stress measures.
+* A new approach for applying an [objective integration](source/materials/lagrangian/ComputeLagrangianObjectiveStress.md) to 
+constitutive models defined in terms of the small (engineering) stress and
+strain to "promote" the response of these models to provide a large
+deformation material response.
+* [Partial compatibility](source/materials/lagrangian/ComputeLagrangianWrappedStress.md) with the current MOOSE materials system.
+* A [new homogenization subsystem](modules/tensor_mechanics/Homogenization.md) for applying cell-average stress or
+deformation constraints to periodic domains, useful for homogenizing the
+response of microstructural representative volume elements.
```

Comment:
```suggestion
- Cartesian versions of a reformulated [updated Lagrangian](source/kernels/lagrangian/UpdatedLagrangianStressDivergence.md) kernel and a new
  [total Lagrangian](source/kernels/lagrangian/TotalLagrangianStressDivergence.md) kernel, both with exact large deformation Jacobians.
- [Stabilization](modules/tensor_mechanics/Stabilization.md) for linear triangular and tetraheadral elements.
- A [new material system](modules/tensor_mechanics/NewMaterialSystem.md), which provides new options for implementing
  material models, including options for defining material models with alternative stress measures.
- A new approach for applying an [objective integration](source/materials/lagrangian/  ComputeLagrangianObjectiveStress.md) to
  constitutive models defined in terms of the small (engineering) stress and strain to "promote" the response of these models
  to provide a large deformation material response.
- [Partial compatibility](source/materials/lagrangian/ComputeLagrangianWrappedStress.md) with the current MOOSE materials system.
- A [new homogenization subsystem](modules/tensor_mechanics/Homogenization.md) for applying cell-average stress or
  deformation constraints to periodic domains, useful for homogenizing the response of microstructural representative volume elements.
```

Built this locally to get the right formatting here. We use dashes to create a bulleted list in our markdown syntax, and multi-line list entries need to have spaces added to them to align with the start of the entry it corresponds to. 

I also noticed that `UpdatedLagrangianKernel` and `TotalLagrangianKernel` did not exist....changed this to `UpdatedLagrangianStressDivergence` and `TotalLagrangianStressDivergence`. Feel free to make further adjustments if I was wrong on that one.